src/main/cpp/v8/JNIV8Array.cpp

```diff
@@ -98,7 +98,7 @@ jobjectArray JNIV8Array::jniGetV8ElementsInRange(JNIEnv *env, jobject obj, jint
                     break;
                 case JNIV8MarshallingError::kOutOfRange:
                     ThrowV8RangeError("value '"+
-                                      JNIV8Marshalling::v8string2string(value->ToString(isolate))+"' is out of range for element element #" + std::to_string(i));
+                                      JNIV8Marshalling::v8string2string(value->ToString(isolate->GetCurrentContext()).ToLocalChecked())+"' is out of range for element element #" + std::to_string(i));
```

Comment:
No need to get the context again; JNIV8Object_PrepareJNICall already defines a local variable "context" that you can just pass in here :)